multidim-interop/go/v0.23/Makefile

```diff
@@ -1,11 +1,27 @@
-image_name := go-v0.23
+SHELL := /usr/bin/env bash
+imageName := go-v0.23
 
+cacheKey=${shell find . -type f | sort | grep -v image.json  | tr "\n" " " | xargs ../../helpers/hashFiles.sh}
```

Comment:
This is a lot of duplicated, non-trivial code. Would it make sense to split out the common parts into a separate script?

Reply:
Happy to make that change in a follow up PR. But I don’t think this is that complicated. This doesn’t actually need the sort call since that’s handled by the helper.

We could also have the helper read from stdin instead of the args and that would remove the `tr`.

Long term though this will go away as all the future versions are defined in the implementation repos and the cache key becomes the commit.